apollo-federation/src/sources/connect/validation/snapshots/validation_tests@invalid_entity_arg_field_args_missing.graphql.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: apollo-federation/src/sources/connect/validation/mod.rs
+expression: "format!(\"{:#?}\", errors)"
+input_file: apollo-federation/src/sources/connect/validation/test_data/invalid_entity_arg_field_args_missing.graphql
+---
+[
+    Message {
+        code: EntityResolverArgumentMismatch,
+        message: "`@connect(entity: true)` on `Query.product` is missing entity resolver arguments.",
```

Comment:
```suggestion
        message: "`Query.product` with `@connect(entity: true) must have arguments.",
```

this will be a little trickier, but we're describing the field, not the connect directive.

maybe we should also link to https://go.apollo.dev/connectors/directives#rules-for-entity-true